### DISCUSSION TITLE
I want to turn on a light when there is a User unread message

### DISCUSSION THREAD

**Discussion by soypat** [slack-go/slack#1136] [Q&A] [answered]
I've setup the App with the following User scopes:
![image](https://user-images.githubusercontent.com/26156425/204062798-54883d59-ab33-49c5-800c-53bc218ba8ac.png)

I'm unsure how to go about reading direct messages. GetConversation fails with "missing_scope".

**Comment by kanata2** [comment, marked as answer]
`GetConversations` method call `conversations.list` API.
This requires some more scopes. 
Please see official doc. https://api.slack.com/methods/conversations.list
